frontend/src/index.js

```diff
@@ -4,11 +4,13 @@ import './index.css';
 import App from './components/app/App';
 import reportWebVitals from './reportWebVitals';
 import { BrowserRouter } from 'react-router-dom';
+import FormsContainer from './components/formsContainer/FormsContainer'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <BrowserRouter>
     <React.StrictMode>
+      <FormsContainer />
```

Comment:
do we need this line here or should we move this inside app.js? I thought the index.js was kept as clean as possible and the actual app was constructed inside app.js